tests/integration/test_sanity.py

```diff
@@ -27,7 +18,19 @@ def test_echo_hello_command(self):
         result = cmd.result()
         self.assertEqual(result, 'hello\n')
 
-    def test_max_output(self):
+    def test_max_output_exceeds_maximum(self):
         cmd = self.client.run('MyHostId', 'bash', '-ce', 'yes')
         with self.assertRaises(CommandAbortedByOverflow):
             cmd.result()
+
+    def test_max_output_per_channel_set(self):
+        max_output_per_channel = 3
+        for val, expected_ret in [('123', '0'), ('1234', 'overflowed')]:
+            cmd = self.client.run(
+                'MyHostId', 'bash', '-ce', 'python -c "print(\'{}\', end=\'\')"'.format(val),
+                max_output_per_channel=max_output_per_channel)
+            ret = get_retcode(self.client.redis, cmd.job_id)
+            self.assertEqual(ret, expected_ret)
+            if expected_ret == 0:
```

Comment:
do we want to validate the output when it's overflowed?

Reply:
if we have overflow,  the stdout key won't exist, so we don't have what to validate